src/components/base-address/factories/address-factory.ts

```diff
@@ -187,21 +187,22 @@ export function useCountryRegions (code: string): boolean {
 }
 
 export function formatAddress (address: AddressIF): AddressIF {
-  address.postalCode = address.postalCode?.toUpperCase() || ''
-  if (address.country === 'CA') {
+  let formattedPostalCode = address.postalCode?.toUpperCase() || ''
```

Comment:
Yes, don't change the passed-in address object. Good 👍

Reply:
Wait. Should you be using `formattedPostalCode` on line 203?

The other thing you could do is move toUpperCase() into line 203 (chaining).

Reply:
This should be fixed. Using formattedPostalCode now on line 203:
https://github.com/bcgov/bcrs-shared-components/blob/4a37dd52699fa636fe23ba61beb0413179c74daf/src/components/base-address/factories/address-factory.ts#L203

Closing the conversation.